server/src/db/migrations/20231101142209_createReviews.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("reviews", (table) => {
+    table.bigIncrements("id");
+    table.bigInteger("movieId").notNullable().unsigned().index().references("movies.id");
+    table.bigInteger("userId").notNullable().unsigned().index().references("users.id");
+    table.string("reviewContent").notNullable();
```

Comment:
consider renaming this column just `content` so that we do not repeat the table name as part of a column name. This means we will access as `review.content` as opposed to repeating ourselves `review.reviewContent`